docs/formats/csup.md

```diff
@@ -1,60 +1,58 @@
-VNG is the "stacked" version of Zed, where the fields from a stream of
-Zed records are stacked into vectors that form columns.
+Super Columnar is a file format based on
+the [super data model](zed.md) where data is stacked to form columns.
 Its purpose is to provide for efficient analytics and search over
-bounded-length sequences of [Super Binary](bsup.md) data that is stored in columnar form.
+bounded-length sequences of [super-structured data](./README.md#2-a-super-structured-pattern) that is stored in columnar form.
 
 Like [Parquet](https://github.com/apache/parquet-format),
-VNG provides an efficient columnar representation for semi-structured data,
-but unlike Parquet, VNG is not based on schemas and does not require
+Super Columnar provides an efficient representation for semi-structured data,
+but unlike Parquet, Super Columnar is not based on schemas and does not require
 a schema to be declared when writing data to a file.  Instead,
-VNG exploits the super-structured nature of Zed data: columns of data
+it exploits the nature of super-structured data: columns of data
 self-organize around their type structure.
 
-## VNG Files
+## Super Columnar Files
 
-A VNG file encodes a bounded, ordered sequence of Zed values.
-To provide for efficient access to subsets of VNG-encoded data (e.g., columns),
-the VNG file is presumed to be accessible via random access
+A Super Columnar file encodes a bounded, ordered sequence of values.
+To provide for efficient access to subsets of Super Columnar-encoded data (e.g., columns),
+the file is presumed to be accessible via random access
 (e.g., range requests to a cloud object store or seeks in a Unix file system)
-and VNG is therefore not intended as a streaming or communication format.
+and is therefore not intended as a streaming or communication format.
 
-A VNG file can be stored entirely as one storage object
+A Super Columnar file can be stored entirely as one storage object
 or split across separate objects that are treated
-together as a single VNG entity.  While the VNG format provides much flexibility
+together as a single Super Columnar entity.  While the format provides much flexibility
 for how data is laid out, it is left to an implementation to lay out data
 in intelligent ways for efficient sequential read accesses of related data.
 
 ## Column Streams
 
-The VNG data abstraction is built around a collection of _column streams_.
+The Super Columnar data abstraction is built around a collection of _column streams_.
 
 There is one column stream for each top-level type encountered in the input where
 each column stream is encoded according to its type.  For top-level complex types,
 the embedded elements are encoded recursively in additional column streams
 as described below.  For example,
-a record column encodes a "presence" vector encoding any null value for
+a [record column](#record-column) encodes a [presence column](#presence-columns) encoding any null value for
```

Comment:
Another spot I'm de-"vector"-ing. Since the related section was already called "presence columns" this seemed like a safe change.